run.service-auth.receive_test: test_auth failed

Note: #10698 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: d1d77c5a323daeeee90a9f40d5efd66a7d7a6b04
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/d4e8521c-d04d-43f3-9705-0655874fb78e), [Sponge](http://sponge2/d4e8521c-d04d-43f3-9705-0655874fb78e)
status: failed

```

__________________________________ test_auth ___________________________________
Traceback (most recent call last):
  File "/workspace/run/service-auth/receive_test.py", line 118, in test_auth
    assert "Hello" in response.content.decode("UTF-8")
AssertionError: assert 'Hello' in 'Error verifying ID token: Could not verify token signature.'
 +  where 'Error verifying ID token: Could not verify token signature.' = ('UTF-8')
 +    where  = b'Error verifying ID token: Could not verify token signature.'.decode
 +      where b'Error verifying ID token: Could not verify token signature.' = .content
```

--- a/run/service-auth/receive_test.py
+++ b/run/service-auth/receive_test.py
@@ -70,7 +70,7 @@ def services():
 
     token = subprocess.run(
         ["gcloud", "auth", "print-identity-token"], stdout=subprocess.PIPE, check=True
-    ).stdout.strip()
+    ).stdout.strip().decode()
 
     yield endpoint_url, token
 

--- a/run/service-auth/receive_test.py
+++ b/run/service-auth/receive_test.py
@@ -66,7 +66,7 @@ def services():
         ],
         stdout=subprocess.PIPE,
         check=True,
-    ).stdout.strip()
+    ).stdout.strip().decode()
 
     token = subprocess.run(
         ["gcloud", "auth", "print-identity-token"], stdout=subprocess.PIPE, check=True
@@ -92,8 +92,8 @@ def services():
 
 
 def test_auth(services):
-    url = services[0].decode()
-    token = services[1].decode()
+    url = services[0]
+    token = services[1]
 
     req = request.Request(url)
     try:

